git-cliff/src/lib.rs

```diff
@@ -676,13 +693,20 @@ pub fn run_with_changelog_modifier(
 
 			// Process the repository.
 			let repository = Repository::init(repository)?;
+
+			// The commit range, used for determining the remote commits to include
+			// in the changelog, doesn't make sense if multiple repositories are
+			// specified. As such, pick the commit range from an arbitrary
+			// repository.
+			commit_range = determine_commit_range(&args, &config, &repository)?;
```

Comment:
Not sure what you mean here by an "arbitrary" repository. Isn't this being run for each repository?

Reply:
It is, but the `commit_range` variable will be updated on each loop iteration, so in practice only the commit range of the last repository specified will be used (which I called "arbitrary" here). I'm not sure how this PR can play nice with the ability to specify commits from multiple repositories, since we only fetch commits from one repository using the remote API. Let me know if you have any suggestions.

Reply:
Ah I see. I think then the comment should be clarified about how this works (i.e. the last one is picked) - otherwise it's not clear to me what arbitrary means.

We can do some tests for multiple repositories... but I'd say don't worry about that for now :)